src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -98,6 +106,16 @@ public static ProviderCultureResult ParseCookieValue(string value)
             var cultureName = potentialCultureName.Substring(_culturePrefix.Length);
             var uiCultureName = potentialUICultureName.Substring(_uiCulturePrefix.Length);
 
+            if (!IsValidCulture(cultureName))
```

Comment:
In that case, let's do it after we do all the null-checks. I don't want us to log things if we ultimately wouldn't do anything with it.